cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail.po.ts

```diff
@@ -21,11 +21,51 @@ export default abstract class ClusterManagerDetailPagePo extends PagePo {
     return this.self().find('.primaryheader h1');
   }
 
+  logsContainer() {
+    return this.self().find('.logs-container');
+  }
+
   kubectlCommandForImported() {
     return this.self().get('code').contains('--insecure');
   }
 
   machinePoolsList() {
     return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
   }
+
+  conditionsList() {
+    return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
+  }
+
+  referredToList() {
+    return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
+  }
+
+  downloadYamlButton() {
+    return this.self().get('[data-testid="sortable-table-download"]');
+  }
+
+  searchFilter() {
+    return this.self().get('.input-sm');
+  }
+
+  firstMachineRow() {
+    return this.self().get('[data-testid="sortable-table-0-row"]');
+  }
+
+  clickConditionsTab() {
+    return this.self().get('[data-testid="btn-conditions"]').click();
+  }
+
+  clickRelatedTab() {
+    return this.self().get('[data-testid="btn-related"]').click();
+  }
+
+  clickLogTab() {
+    return this.self().get('[data-testid="btn-log"]').click();
+  }
+
+  clickMachinesTab() {
+    return this.self().get('[data-testid="btn-node-pools"]').click();
+  }
```

Comment:
these should use the `TabbedPo` (cypress/e2e/po/components/tabbed.po.ts) and that way they could be consolidated into one method like this

``` 
selectTab(options: TabbedPo, selector: string) {
    options.clickTabWithSelector(selector);

    return this;
  }
```

Reply:
i didn't mean to include 52-54 in this comment sry